packages/core/src/hooks/useEthers.test.tsx

```diff
@@ -58,6 +64,25 @@ describe('useEthers', () => {
     expect(result.current.error?.toString()).to.include(`Unsupported chain id: ${SECOND_TEST_CHAIN_ID}`)
   })
 
+  it('throws error if trying to use not configured network', async () => {
+    const notConfiguerdNetwork = await createMockProvider({ chainId: Localhost.chainId })
+    const { result, waitForCurrent } = await renderDAppHook(
+      () => {
+        const { activate } = useEthers()
+        useEffect(() => {
+          void activate(notConfiguerdNetwork.provider)
+        }, [])
+
+        return useEthers()
+      },
+      { config }
+    )
+
+    await waitForCurrent((val) => !!val.error)
+    expect(result.current.error).not.to.be.undefined
+    expect(result.current.error?.toString()).to.include(`Not configured chain id: ${Localhost.chainId}`)
```

Comment:
Can we have test for both not configured and not supported chains?

Reply:
There are different error messages for both cases, but we can only check if the error occured. Should we?

Reply:
I meant 2 tests